build/scripts/GitHub/GitHubPullRequest.class.psm1

```diff
@@ -54,12 +52,31 @@ class GitHubPullRequest {
 
         $issueIds = @()
         foreach($match in $issueMatches.Matches) {
-            $issueIds += $match.Groups[1].Value
+            $issueIds += $match.Groups[2].Value
         }
 
         return $issueIds
     }
 
+    [int[]] GetLinkedADOWorkitems() {
+        if(-not $this.PullRequest.body) {
+            return @()
+        }
+
+        $workitemRegex = "(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved) AB#(\d+)" # e.g. "AB#1234"
+        $workitemMatches = Select-String $workitemRegex -InputObject $this.PullRequest.body -AllMatches
```

Comment:
The logic below seems to be the same as the one after line 47. Let's extract the repeat code in a common `GetLinkedItemIDs` or something like that.